### PR TITLE
Deposit contract script additions

### PR DESCRIPTION
**Pull request by yglukhov**
- Added `deploy`, `drain` and `sendEth` subcommands.
- Added `--privateKey` argument.

### CODE REVIEW

File: beacon_chain/deposit_contract.nim [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
...0208501600060046012f15050805182019150506000601860208206610b800160208284011115610f3357600080fd5b602080610ba082610700600060046015f1505081815280905090509050601880602084610c0001018260208501600060046014f150508051820191505061084051602082610c0001015260208101905080610c0052610c009050602060c0825160208401600060025af1610fa657600080fd5b60c0519050602082610c8001015260208101905080610c8052610c809050602060c0825160208401600060025af1610fdd57600080fd5b60c0519050610ae0526001805460018254011015610ffa57600080fd5b6001815401815550600154610d0052610d2060006020818352015b60016001610d005116141561104a57610ae051610d20516020811061103957600080fd5b600060c052602060c02001556110de565b6000610d20516020811061105d57600080fd5b600060c052602060c0200154602082610d40010152602081019050610ae051602082610d4001015260208101905080610d4052610d409050602060c0825160208401600060025af16110ae57600080fd5b60c0519050610ae052610d0060026110c557600080fd5b60028151048152505b8151600101808352811415611015575b5050005b639890220b60005114156111165734156110fb57600080fd5b600060006000600030316002546000f161111457600080fd5b005b638ba35cdf600051141561113d57341561112f57600080fd5b60025460005260206000f350005b5b60006000fd5b6101836112c7036101836000396101836112c7036000f3"
```
Comment (arnetheduck) [inline beacon_chain/deposit_contract.nim]: can we `slurp` this?

Comment (yglukhov) [reply beacon_chain/deposit_contract.nim]: Sure, just didn't see a reason to do that yet.

File: beacon_chain/deposit_contract.nim [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+contract(Deposit):
+  proc drain()
+
+proc deployContract*(web3: Web3, code: string): Future[Address] {.async.} =
+  let provider = web3.provider
+  var code = code
+  if code[1] notin {'x', 'X'}:
+    code = "0x" & code
+  var tr: EthSend
```
Comment (arnetheduck) [inline beacon_chain/deposit_contract.nim]: technically, this has the classic `EthSend(source: ...)` syntax issue - though it's probably a leftover from the old file

Comment (yglukhov) [reply beacon_chain/deposit_contract.nim]: Fixed

**Pull request merged**